content/docs/studio/troubleshooting.md

```diff
@@ -250,6 +251,18 @@ repository used DVC 1.0. The format of lock files used in DVC 1.0 were
 deprecated in the DVC 2.0 release. Upgrading to the latest DVC version will
 resolve this issue for any future commits in your repository.
 
+## Project does not reflect updates in the Git repository
```

Comment:
@tapadipti tbh, it's not very helpful for users I think. Okay I got a high level explanation, I understand better what's going on, what are the practical things I can do?

- E.g. on BB we know that users need to be a repo admin - it'a common gotcha
- On all systems we probably have a way to go an check if hook is there and if not ask us, something else?
- Is there anything we can do on the Studio side to check it quickly, get more information why it's not working?